adr/adr-template.md

```diff
@@ -0,0 +1,17 @@
+# ADR <Four digit number>: <Title>
+
+## Status
+
+The status of this ADR. It can be either Proposed, Accepted, Rejected, Deprecated, Superseded.
+
+## Context
+
+What is the issue that we're seeing that is motivating this decision or change?
+
```

Comment:
"Context" should probably include a link to the discussion thread that led to the ADR.